clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/gvcf/TransferGvcfV1Metadata.scala

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.clio.transfer.model.gvcf
+
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.transfer.model.TransferMetadata
+import org.broadinstitute.clio.util.model.DocumentStatus
+
+case class TransferGvcfV1Metadata(analysisDate: Option[OffsetDateTime] = None,
+                                  contamination: Option[Float] = None,
+                                  documentStatus: Option[DocumentStatus] = None,
+                                  gvcfMd5: Option[String] = None,
+                                  gvcfPath: Option[String] = None,
+                                  gvcfSize: Option[Long] = None,
+                                  gvcfIndexPath: Option[String] = None,
+                                  gvcfMetricsPath: Option[String] = None,
+                                  pipelineVersion: Option[String] = None,
+                                  notes: Option[String] = None)
+    extends TransferMetadata[TransferGvcfV1Metadata] {
+
+  override def pathsToMove: Seq[String] =
+    Seq.concat(gvcfPath, gvcfIndexPath, gvcfMetricsPath)
+
+  override def rebaseOnto(destination: String): TransferGvcfV1Metadata = {
+    assert(destination.endsWith("/"))
+
+    this.copy(
+      gvcfPath = gvcfPath.map(rebaseOnto(_, destination)),
+      gvcfIndexPath = gvcfIndexPath.map(rebaseOnto(_, destination)),
+      gvcfMetricsPath = gvcfMetricsPath.map(rebaseOnto(_, destination))
+    )
+  }
+
+  override def setSinglePath(destination: String): TransferGvcfV1Metadata = {
+    assert(pathsToMove.length == 1)
```

Comment:
I'm unclear how this can ever be true? Is there a subclass that overrides `pathsToMove`?

Reply:
This will be true if we run the delete CLP on a gvcf key (which will delete the gvcf and index, but not metrics), and then run the move CLP on the same key. Not sure if that'll ever happen, but I didn't want to rule it out either since it's not much more code to support.